app/controllers/admin/stolen_bikes_controller.rb

```diff
@@ -21,8 +21,8 @@ def approve
   end
 
   def regenerate_alert_image
-    @bike.current_stolen_record.generate_alert_image!
-    redirect_to edit_admin_stolen_bike_url(@bike), notice: "Premium alert images regenerated."
+    @bike.current_stolen_record.generate_alert_image
+    redirect_to edit_admin_stolen_bike_url(@bike), notice: "Promoted alert images regenerated."
```

Comment:
Good catch

Reply:
Actually - there are still a number of places that _"premium"_ is being used in this code. Are you willing to replace all of them? Could be a separate PR.

Reply:
Good catch thanks! Updated in b8b0fd45